toto_client/toto_client.py

```diff
@@ -214,6 +214,24 @@ def search_term(self, search_term, tags = None):
                             dataType
                             pageNumber
                             pageIndexes
+                            regs: datas(tagName: "regs") {
+                                id
+                                dataType
+                                pageNumber
+                                text
+                            }
+                            effects: datas(tagName: "effects") {
+                                id
+                                dataType
+                                pageNumber
+                                text
+                             }
+                            sub_evidence: datas(tagName: "sub_evidence") {
+                                id
+                                dataType
+                                pageNumber
+                                text
+                            }
```

Comment:
This change is already implemented in lines 199-207 above for any desired tag using the `tags` parameter. You would need to call the function as below to implement the same query:

```python
toto_client.search_term(search_term="text searched for", tags=["regs", "effects", "sub_evidence"])
```